src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/XASdkTests.cs

```diff
@@ -504,7 +505,7 @@ public void DotNetBuild (string runtimeIdentifiers, bool isRelease, bool aot)
 		}
 
 		[Test]
-		public void SupportedOSPlatformVersion ([Values (21, 30)] int minSdkVersion)
+		public void SupportedOSPlatformVersion ([Values (21, 31)] int minSdkVersion)
```

Comment:
Maybe we should leave this one as API 30 and leave a comment why:
```
--min-api 31
```
This causes the latest version of d8 to print:
```
D8 An API level of 31 is not supported by this compiler. Please use an API level of 30 or earlier 
```
And it seems like we have the latest:
https://mvnrepository.com/artifact/com.android.tools/r8

Reply:
You marked this as resolved but did not make the change you suggested.  This seems to be the only remaining test failure on this PR.  Should we commit your suggestion?

Reply:
Somehow a commit got lost (or I didn't push it).

I still had the commit locally, I pushed it.